endercore/src/main/resources/META-INF/mods.toml

```diff
@@ -20,7 +20,7 @@ modId="forge" #mandatory
 # Does this dependency have to exist - if not, ordering below must be specified
 mandatory=true #mandatory
 # The version range of the dependency
-versionRange="[37,)" #mandatory
+versionRange="[38,)" #mandatory
```

Comment:
maybe also change versionRange to include the secure Version so enderCore and other mods will prohibit to load insecured (and we can ensure that newer forge features we might use are present on the client)

Reply:
At some point in the future a specific minimum version of Forge and Endercore will be required to fix an issue, I agree with agnor